lib/storage/index.js

```diff
@@ -131,6 +131,18 @@ function Storage(config) {
  * var storage = gcloud.storage();
  * var albums = storage.bucket('albums');
  *
+ * //-
+ * // Make a bucket's contents publicly readable.
```

Comment:
Oops, good catch on my good catch! I didn't actually realize they were different. You should definitely clarify whether it makes new objects public or not which I think is the standard case here. The use case that I'm imagining is a website asset storage bucket.